tests/test_logger.py

```diff
@@ -20,10 +20,10 @@ def tearDown(self):
 
     def test_emit(self):
         with mock.patch("iquip.apps.logger._Signaller.signal") as mocked_signal:
-            app = logger.LoggerApp(name="name", parent=QObject())
-            app.setLevel("DEBUG")
+            app = logger.LoggerApp(name="name", logFilePath="", parent=QObject())
```

Comment:
It looks better to assign a non-empty string to `logFilePath`.
Is it possible?

Reply:
If I added `logFilepath = "test"`, it said directory `\\iquip\\test\\log_230821-144124.log'` is not exist.

Reply:
Oh, I see. Then if we set `logFilePath` to an empty string, where is the log file created, in the iquip directory or the called directory?

Reply:
Also, does the unittest make a log file? If so, the file should be removd after the test.